Dockerfile

```
@@ -72,66 +73,65 @@ ENV PATH="${CONDA_PATH}/bin:$PATH" \
     CPATH="${CONDA_PATH}/include:$CPATH" \
     PYTHONNOUSERSITE=1
 
-RUN ${CONDA_PATH}/bin/conda install -c conda-forge -c anaconda \
-                     python=3.8 \
-                     attrs=21.2 \
-                     codecov=2.1 \
-                     colorclass \
-                     coverage=6.0 \
-                     curl \
-                     datalad=0.15 \
-                     dipy=1.4 \
-                     flake8 \
-                     git \
-                     git-annex=*=alldep* \
-                     graphviz=2.49 \
-                     h5py=3.2 \
-                     indexed_gzip=1.6 \
-                     jinja2=2.11 \
-                     libxml2=2.9 \
-                     libxslt=1.1 \
-                     lockfile=0.12 \
-                     "matplotlib>=3.3,<4" \
-                     mkl=2021.3 \
-                     mkl-service=2.4 \
-                     nibabel=3.2 \
-                     nilearn=0.8 \
-                     nipype=1.6 \
-                     nitime=0.9 \
-                     nodejs=16 \
-                     numpy=1.22 \
-                     packaging=21 \
-                     pandas=1.2 \
-                     pandoc=2.14 \
-                     pbr \
-                     pip=21.3 \
-                     pockets \
-                     psutil=5.8 \
-                     pydot=1.4 \
-                     pydotplus=2.0 \
-                     pytest=6.2 \
-                     pytest-cov=3.0 \
-                     pytest-env=0.6 \
-                     pytest-xdist \
-                     pyyaml=5.4 \
-                     requests=2.26 \
-                     scikit-image=0.19 \
-                     scikit-learn=1.0 \
-                     scipy=1.8 \
-                     seaborn=0.11 \
-                     setuptools=58.2 \
-                     sphinx=4.2 \
-                     sphinx_rtd_theme=1.0 \
-                     "svgutils>=0.3.4,<0.4" \
-                     toml=0.10 \
-                     traits=6.2 \
-                     zlib=1.2 \
-                     zstd=1.5; sync && \
+COPY condarc /root/.condarc
+
+RUN ${CONDA_PATH}/bin/conda install \
```

I tested yesterday with the following:
```suggestion
RUN ${CONDA_PATH}/bin/conda install mamba -n base \
+    && mamba install -y \
```

and did not really see much if any speedup

---

I'll add in mamba just to have the option to help with package resolution in subsequent images